e2e/nuxt/jest.config.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable */
+export default {
```

Comment:
Also need to add an entry to `create-project-utils.ts` to intall `@nx/nuxt` package for e2e tests.